packages/google_sign_in/google_sign_in_web/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.12.4+1
+* Fixed README.md typo
```

Comment:
Please follow the changeliog style guide linked from the PR checklist.

Reply:
I think I need to change version to *NEXT*.  
Should I revert the version change in pubspec too?

Reply:
I recommend reading the FAQ section of the linked documentation about READMEs and version updates. Also, you haven't addressed the CHANGELOG message style issues raised in the comment you are replying to.

Please read the documentation linked from the PR checklist; the reason we have the checklist is to provide guidance and links to relevant documentation.

Reply:
This still doesn't use the correct verb form, and still doesn't have a period, so none of the style issues have been resolved.